src/lib/sw_engine/tvgSwRle.cpp

```diff
@@ -665,6 +664,82 @@ SwSpan* _intersectSpansRegion(const SwRleData *clip, const SwRleData *targetRle,
     return out;
 }
 
+SwSpan* _intersectMaskRegion(const SwRleData *clip, const SwRleData *targetRle, SwSpan *outSpans, uint32_t spanCnt)
```

Comment:
Question: how's functionally different with _intersectSpansRegion()?

Reply:
this function works differently then _intersectSpansRegion(), here shape is overlaped by outSpans without data defined as mask